layers/meta-balena-compulab/recipes-kernel/linux/linux-imx/0001-Add-support-for-NXP-PCA9956B-LED-controller.patch

```diff
@@ -0,0 +1,819 @@
+From 4697aa5874e38bd4a82c0f134bedee6c8d2032e4 Mon Sep 17 00:00:00 2001
```

Comment:
Please add in the git log the revision from https://github.com/BeastRoms-Devices/kernel_realme_RMX1927/ that you used